components/statuscake/sources/new-uptime/new-uptime.mjs

```diff
@@ -0,0 +1,62 @@
+import statuscake from "../../statuscake.app.mjs";
+
+export default {
+  name: "New Uptime",
+  version: "0.0.1",
+  key: "statuscake-new-uptime",
+  description: "Emit new event for each uptime created.",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    statuscake,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      static: {
+        intervalSeconds: 15 * 60, // 15 minutes
+      },
+    },
+  },
+  methods: {
+    emitEvent(data) {
+      this.$emit(data, {
+        id: data.id,
+        summary: `New uptime with id ${data.id}`,
+        ts: new Date(),
+      });
+    },
+  },
+  hooks: {
+    async deploy() {
+      const uptimes = await this.statuscake.getUptimes({
+        uptimeId: this.uptimeId,
+        params: {
+          limit: 10,
+        },
+      });
+
+      uptimes.reverse().forEach(this.emitEvent);
+    },
+  },
+  async run() {
```

Comment:
Again, if it is possible(e.g. if new IDs are always greater than older ones), can we store latest IDs(or some other info) to compare and differentiate new uptimes? Because it seems that when timer ticks, all uptimes will be fetched.